jitbuilder/compile/ResolvedMethod.cpp

```diff
@@ -127,13 +127,13 @@ JitBuilder::ResolvedMethod::computeSignatureChars()
       {
       name = _parmTypes[p]->getSignatureName();
       len = static_cast<uint32_t>(strlen(name));
-      strncpy(_signatureChars+s, name, len);
+      strncpy(_signatureChars+s, name, 64-s);
```

Comment:
Since you're modifying this code anyway, I think the 64 should be replaced with a literal constant (either a macro or a static const field). There's references to `64` and `62` wrt to `_signatureChars` that I could find grepping through the code.

Reply:
Yes, all right.